exporter/signalfxexporter/README.md

```diff
@@ -0,0 +1,41 @@
+# SignalFx Metrics Exporter
+
+How to send metrics to SignalFx.
+
+The following configuration options are required:
+
+- `access_token` (no default): AccessToken is the authentication token provided
+by SignalFx or another backend that supports the SAPM proto.
```

Comment:
```suggestion
by SignalFx or another backend that can receive SignalFx metrics.
```

Reply:
alternatively, just remove the part starting after "by SignalFx"

Reply:
Removed rest